app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,9 @@
+class NotificationsController < ApplicationController
+  def disable 
+    if current_user
```

Comment:
This should be improved so that the user can still turn off notifications - even if they are logged out. Provide a link to login etc